reference/constraints/_empty-values-are-valid.rst.inc

```diff
@@ -0,0 +1,7 @@
+.. note::
+
+    As with most of the other constraints, ``null`` values and empty strings
+    are considered valid values. Otherwise, in addition to validating this value,
+    you would also be requiring it, making it impossible to be optional. That's
+    why it's common to combine this constraint with
+    :doc:`NotBlank </reference/constraints/NotBlank>`.
```

Comment:
I thought if we should maybe reword this a bit:

> As with most of the other constraints, `null` values and empty strings are considered valid values. This is to allow them to be optional values. If the values is mandatory, a common solution is to combine this constraint with `NotBlank`.

Reply:
I think that your wording is better. Could you please create a pull request for it? Thanks a lot!

Reply:
here we go: #10418